file_paths.md

```diff
@@ -1,24 +1,341 @@
+            bucket                        # :bucketdir
+            client_yaml                   # :clientyamldir
+            client_data                   # :client_datadir
+            clientbucket                  # :clientbucketdir
+            devices                       # :devicedir
+            facts.d                       # :pluginfactdest (pluginsync'ed)
+            lib                           # :libdir
+            facts                         # used to generate :factpath
+            puppet-module                 # :module_working_dir
+            reports                       # :reportdir
+            server_data                   # :server_datadir
+            state                         # :statedir
+            yaml                          # :yamldir
+        include *
+            facter *
+            openssl *
+        lib *
+            libaugeas.so *
+            libcrypto.so *
+            libfacter.so *
+            libruby.so *
+            libssl.so *
+            ruby *
+                vendor_ruby *             # ruby code
+                    facter.rb *
+                    hiera.rb *
+                    puppet.rb *
+            virt-what *
+                virt-what-cpuid-helper *
+        modules *                         # system modulepath
+        share *
+            augeas *
+            man *
+            vim *
+            locale *
+                config.yaml *             # configuration file used by the gettext-setup gem
+                <lang_COUNTRY> *
+                    LC_MESSAGES *
+                        <project>.mo *    # One directory for each supported locale,
+                                          # containing the MO files for each project.
+                                          # Example for English:
+                                          # /opt/puppetlabs/puppet/share/locale/en_US/LC_MESSAGES/puppet.mo
+        ssl *
+        vendor_modules *                  # :vendormoduledir
+        VERSION                           # puppet-agent package version
+
+    /opt/puppetlabs/pxp-agent *
+        modules *
+            pxp-module-puppet *
+        spool *                            # directory containing results of pxp-agent modules
+        tasks-cache *                      # directory containing a cache of files for all downloaded tasks
+
+    /var/log/puppetlabs *
+        puppet *                          # :logdir                      /var/lib/puppet/log
+            puppet.log                    # not enabled by default
+        pxp-agent *
+            pxp-agent.log                 # enabled by default
+            pcp-access.log                # not enabled by default
+
+    /var/run/puppetlabs *                 # :rundir                      /var/lib/puppet/run
+        agent.pid                         # :pidfile
+        pxp-agent.pid
+
+# puppet-agent 6 (Windows)
+
+On Windows:
+
+- The installation path defaults to `C:\Program Files\Puppet Labs\Puppet`
```

Comment:
I think I know what you're getting at here, but the wording is off. The default path used by Agent is `[Program Files]\Puppet Labs`.  Thing is though I'm not sure how to word that better. Perhaps;

> The installation path defaults to `Puppet Labs\Puppet` under the `Program Files` directory.  For example on a default Windows installation this would be `C:\Program Files\Puppet Labs\Puppet`